packages/fx-core/templates/plugins/resource/function/bicep/module.template.bicep

```diff
@@ -0,0 +1,24 @@
+
+module functionProvision '{{PluginOutput.fx-resource-function.Modules.functionProvision.Path}}' = {
+  name: 'functionProvision'
+  params: {
+    functionAppName: function_webappName
+    functionServerfarmsName: function_serverfarmsName
+    functionStorageName: function_storageName
+    aadClientId: aadClientId
+    aadClientSecret: aadClientSecret
+    m365TenantId: m365TenantId
+    applicationIdUri: applicationIdUri
+    oauthAuthorityHost: m365OauthAuthorityHost
```

Comment:
also add `m365` to the beginning of parameter?

Reply:
sure